x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/lens_configuration_flyout.tsx

```diff
@@ -128,6 +135,30 @@ export function LensEditConfigurationFlyout({
     return () => s?.unsubscribe();
   }, [dispatch, output$, layers]);
 
+  useEffect(() => {
```

Comment:
how is this needed here?

Reply:
It is because you need to initialize it, otherwise the accordion is not visible